Source/WebCore/layout/formattingContexts/inline/InlineItem.h

```diff
@@ -109,6 +114,11 @@ inline void InlineItem::setWidth(InlineLayoutUnit width)
     m_hasWidth = true;
 }
 
+inline void InlineItem::setOffset(float offset)
+{
+    m_offset = offset;
+}
+
```

Comment:
I assume this is not an offset on _any_ type of inline item. please assert that we put this on the correct type of inline items.